src/pages/welcome.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="shortcut icon" href="favicon.ico">
+    <link rel="stylesheet" href="app.min.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
+    <meta name="theme-color" content="#002275" />
+    <meta name="description" content="Try Firefox Private Network, a Firefox browser extension that protects you from hackers and trackers when you’re on public Wi-Fi with your laptop." />
+    <meta property="og:description" content="Try Firefox Private Network, a Firefox browser extension that protects you from hackers and trackers when you’re on public Wi-Fi with your laptop." />
+    <meta property="og:image" content="https://private-network.firefox.com/img/private-network-social.png" />
+    <meta property="og:title" content="Firefox Private Network" />
+    <meta property="og:url" content="https://private-network.firefox.com/" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:description" content="Try Firefox Private Network, a Firefox browser extension that protects you from hackers and trackers when you’re on public Wi-Fi with your laptop." />
+    <meta name="twitter:title" content="Firefox Private Network" />
+    <meta property="robots" content="noindex"/>
+    <title>Firefox Private Network</title>
+  </head>
+  <body>
+    <header>
+      <div class="flx flx-row h-padding v-padding header-bg">
+        <div class="logo"></div>
+        <div class="logotype"></div>
+      </div>
+    </header>
+    <main class="section document">
+      <div class="row">
+        <div class="document-content">
+          <h1 class="document-headline ff-Met">Welcome to the Firefox Private Network Beta</h1>
+          <p class="lead">Thanks for participating in this limited-time, free beta of Firefox Private Network. Firefox Private Network lets you use public Wi-Fi more safely and securely. It does this by encrypting the web addresses you visit and the data you send to most websites. This helps protect your web browsing from hackers, websites, and trackers.</p>
```

Comment:
Per https://github.com/mozilla/secure-proxy/issues/617#issue-492985348

Correct String | Incorrect String | Location
-- | -- | --
It does this by encrypting the web addresses you visit. | It does this by encrypting the web addresses you visit and the data you send to most websites. | 1st paragraph